tests/unit/setup.js

```diff
@@ -45,3 +46,7 @@ global.beforeEach(() => {
   });
   global.localVue = localVue;
 });
+
+global.afterEach(() => {
```

Comment:
It is good practice to reset test state in before each hook.

Reply:
Ok, are you going to change that in RIS as well then? 😀

Reply:
👍